NoteAId-FE/app/components/Card.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
```

Comment:
I noticed some of these components are not being used in the app, so we should get rid of them. There are other ones such as `AuthNavigator`, `FeedNavigator`, `NewListingButton`, etc...

Reply:
All the misc files have been removed now Mahdi!